assets/armsnippets.jsonc

```diff
@@ -140,6 +140,122 @@
+            "                       }",
+            "                   }",
+            "             }",
+            "        ],",
+            "        \"frontendIPConfigurations\": [",
+            "            {",
+            "                \"name\": \"${7:appGatewayFrontendIP}\",",
+            "                \"properties\": {",
+            "                    \"PublicIPAddress\": {",
+            "                        \"id\": \"[resourceId('Microsoft.Network/publicIPAddresses', '${3:publicIPAddress1}')]\"",
+            "                    }",
+            "                }",
+            "            }",
+            "          ],",
+            "          \"frontendPorts\": [",
+            "            {",
+            "                \"name\": \"${8:appGatewayFrontendPort}\",",
+            "                \"properties\": {",
+            "                    \"Port\": ${9:80}",
+            "                }",
+            "            }",
+            "          ],",
+            "          \"backendAddressPools\": [",
+            "            {",
+            "                \"name\": \"${10:appGatewayBackendPool}\",",
+            "                \"properties\": {",
+            "                    \"BackendAddresses\": [",
+            "                      {",
+            "                          \"IpAddress\": \"${11:backendIpAddress1}\"",
+            "                      },",
+            "                      {",
+            "                          \"IpAddress\": \"${12:backendIpAddress2}\"",
+            "                      }",
+            "                    ]",
+            "                }",
+            "            }",
+            "          ],",
+            "          \"backendHttpSettingsCollection\": [",
+            "             {",
+            "                \"name\": \"${13:appGatewayBackendHttpSettings}\",",
+            "                \"properties\": {",
+            "                    \"Port\": ${14:80},",
+            "                    \"Protocol\": \"${15|Http,Https|}\",",
+            "                    \"CookieBasedAffinity\": \"Disabled\"",
+            "                }",
+            "            }",
+            "          ],",
+            "          \"httpListeners\": [",
+            "            {",
+            "                \"name\": \"${16:appGatewayHttpListener}\",",
+            "                \"properties\": {",
+            "                  \"FrontendIPConfiguration\": {",
+            "                    \"Id\": \"[resourceId('Microsoft.Network/applicationGateways/frontendIPConfigurations', '${1:applicationGatewayName}', '${8:appGatewayFrontendIP}')]\"",
+            "                  },",
+            "                  \"FrontendPort\": {",
+            "                    \"Id\": \"[resourceId('Microsoft.Network/applicationGateways/frontendPorts',  '${1:applicationGatewayName}', '${9:appGatewayFrontendPort}')]\"",

```

Comment:
I believe the tabstop here is also incorrect.

${9:appGatewayFrontendPort} > ${8:appGatewayFrontendPort}